bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,338 @@
+
+                    # Send random party emojis
+                    party = [random.choice([':partying_face:', ':tada:']) for _ in range(random.randint(1, 10))]
+                    await channel.send(" ".join(party))
+
+                else:
+                    # no participants in current iteration
+                    await channel.send('Hmm... Looks like no one participated! :cry:')
+
+            async with self.checking_messages:  # Acquire the lock to delete the messages
+                self.messages = {}  # reset the messages
+
+        # send the next name
+        self.word = f"{random.choice(self.first_names)} {random.choice(self.last_names)}"
+
+        await channel.send("Let's move on to the next name!\nAnd the next name is...\n"
+                           f"**{self.word}**!\nTry to spookify that... :smirk:")
+
+        self.poll = False  # accepting responses
+
+    async def get_responses_list(self, final: bool = False) -> Embed:
+        """Returns an embed containing the responses of the people."""
+        channel = await self.get_channel()
+
+        embed = Embed(
+            color=Colours.red,
+            title="",
+            description=""
+        )
+
+        if self.messages:
+            if final:
+                embed.title = "Spook Name Rate is about to end!"
+                embed.description = ("This Spook Name Rate round is about to end in 2 hours! You can review "
+                                     "the entries below! Have you rated other's words?")
+            else:
+                embed.title = "All the spookified names!"
+                embed.description = "See a list of all the entries entered by everyone!"
+        else:
+            embed.title = "No one has added an entry yet..."
+
+        for message_id, data in self.messages.items():
+
+            embed.add_field(
+                name=data['author'],
+                value=f"[{data['word']}](https://discord.com/channels/{Client.guild}/{channel.id}/{message_id})",
+            )
+
+        return embed
+
+    async def get_channel(self) -> TextChannel:
+        """Gets the seasonalbot-channel after waiting until ready."""
+        await self.bot.wait_until_ready()
+        return self.bot.get_channel(self.channel_id)
+
+    @staticmethod
+    def load_json(file: Path) -> Dict[str, str]:
+        """Loads a JSON file and returns its contents."""
+        with file.open('r', encoding='utf-8') as f:
+            return json.load(f)
+
+    @staticmethod
+    def in_month() -> bool:
+        """Returns whether running in the limited month."""
+        return datetime.utcnow().month == month_to_limit_to
+
+    def cog_check(self, ctx: Context) -> bool:
+        """A command to check whether the command is being called in October."""
+        if self.in_month():
+            return True
+        else:
+            raise InMonthCheckFailure(f"You can only use these commands in {calendar.month_name[month_to_limit_to]}!")
```

Comment:
```suggestion
        if not self.in_month():
            raise InMonthCheckFailure(f"You can only use these commands in {calendar.month_name[month_to_limit_to]}!")
        return True  
```
failing fast is often easier to read